nilearn/input_data/nifti_labels_masker.py

```diff
@@ -286,6 +286,15 @@ def transform_single_imgs(self, imgs, confounds=None):
             shape: (number of scans, number of labels)
 
         """
+
+        if float(self.n_jobs) < -1 or not float(self.n_jobs).is_integer() or \
```

Comment:
IIUC the check is repeated here to avoid doing the resampling if the ExtractionFunctor init is going to fail, is that right?

Reply:
Correct!